app/helpers/versions_helper.rb

```diff
@@ -2,35 +2,80 @@ module VersionsHelper
   ##
   # Groups functions related to change description
   ##
-  def link_if_alive(version, link_text, link_url)
-    version.item ? link_to(link_text, link_url) : link_text
+  def link_if_alive(version, link_text, link_url, conference)
+    version.item && conference ? link_to(link_text, link_url) : "#{link_text} with ID #{version.item_id}"
+  end
+
+  def link_to_conference(conference_id)
+    return 'deleted conference' if conference_id.nil?
+
+    conference = Conference.find_by(id: conference_id)
+    if conference
+      link_to conference.short_title,
+              edit_admin_conference_path(conference.short_title)
+    else
+      short_title = current_or_last_object_state('Conference', conference_id).try(:short_title) || ''
+      " #{short_title} with ID #{conference_id}"
+    end
+  end
+
+  def link_to_user(user_id)
```

Comment:
Oops! I will delete the one in pathshelper as we don't use it anywhere else, but versions.